lxd/containers_post.go

```diff
@@ -201,117 +201,104 @@ func createFromNone(d *Daemon, req *containerPostReq) Response {
 }
 
 func createFromMigration(d *Daemon, req *containerPostReq) Response {
-	if req.Source.Mode != "pull" {
-		return NotImplemented
+	push := false
```

Comment:
I agree that we probably still want to return an error if Mode isn't "pull" or "push".